src/controller/CHIPDeviceController.cpp

```diff
@@ -271,11 +262,11 @@ CHIP_ERROR ChipDeviceController::ConnectDeviceWithoutSecurePairing(NodeId remote
     return CHIP_NO_ERROR;
 }
 
-CHIP_ERROR ChipDeviceController::EstablishSecureSession()
+CHIP_ERROR ChipDeviceController::EstablishSecureSession(NodeId peer)
```

Comment:
Add `EstablishUnsecureSession` instead?

Reply:
We already have `ConnectDeviceWithoutSecurePairing` so I would suggest using mIsSecure as an internal state.

Some other thoughts: `EstablishSecureSession` -> `EstablishSession` (remove word "Secure")